css/styles.css

```diff
@@ -37,3 +38,49 @@ button{
   color: white;
   background-color: rgb(105, 37, 105);
 }
+
+.margin-top{
+  margin-top: 40px;
+}
+
+.entry-form{
+  display: none;
+}
+
+.indent{
+  padding-left: 0%;
+}
+
+.anchor{
+  text-decoration: none;
+  color: white;
+}
+
+nav{
+  margin-top: 10px;
+}
+
+.entries-tag{
+  border: none;
+}
+
+.gap{
+  gap: 30px;
+}
+
+.text-half-row{
+  display: flex;
+  flex-basis: 50%;
+  justify-content: space-around;
+}
```

Comment:
This looks like you are trying to combine a row, column, and what should be a utility class for `justify-content`.